README.md

```diff
@@ -4,7 +4,7 @@
 
 
 
------        
+-----
```

Comment:
my editor stripped whitespace at the end of lines in this, let me know if that's an issue and i'll revert!